cl/search/templates/includes/calendar-and-relative-date-fields.html

```diff
@@ -0,0 +1,82 @@
+<div class="form-group date-filter-mode">
+  <label>Date Filed:</label>
+  <label class="radio-inline">
+    <input type="radio" name="date-mode" value="calendar"
+      {% if not request.GET.filed_after or "/" in request.GET.filed_after %}
+        checked
+      {% endif %}
+    > Calendar
+  </label>
+  <label class="radio-inline">
+    <input type="radio" name="date-mode" value="relative"
+      {% if request.GET.filed_after and not "/" in request.GET.filed_after %}
+        checked
+      {% endif %}
+    > Relative
+  </label>
+</div>
+
+<div class="date-calendar-fields
+  {% if request.GET.filed_after and not "/" in request.GET.filed_after %}
+    htmx-hidden-indicator
```

Comment:
We should probably use the Bootstrap `hidden` class here, as this one is intended for HTMX loading indicators. Was there a particular reason you opted for this class?

Reply:
Oh yeah, the use of `htmx-hidden-indicator` was part of an old approach where I just tweaked the element’s display in JS. But then I switched to toggling the class. So yeah, it makes sense to use `hidden`. I've made the change.